libkbfs/key_manager_test.go

```diff
@@ -182,8 +182,34 @@ func expectRekey(config *ConfigMock, bh tlf.Handle, numDevices int,
 		gomock.Any()).AnyTimes()
 }
 
-func TestKeyManagerPublicTLFCryptKey(t *testing.T) {
-	runTestOverMetadataVers(t, testKeyManagerPublicTLFCryptKey)
+func TestKeyManager(t *testing.T) {
```

Comment:
Please place this main runner func at the bottom of each file, so it refers to things defined above it.  Same for the other test files.